packages/babel-plugin-transform-function-sent/test/fixtures/function-sent/after-yield/actual.js

```diff
@@ -0,0 +1,5 @@
+(function* () {
```

Comment:
I don't think we do have anything like this currently unfortunately.

I guess it may be fine to leave that as a future optimization, as long as the function is only wrapped if `function.sent` is in use.